python/dask_cudf/dask_cudf/tests/utils.py

```diff
@@ -19,8 +19,9 @@
 
 
 def _make_random_frame(nelem, npartitions=2, include_na=False):
+    rng = np.random.default_rng(seed=None)
```

Comment:
I still see a few `seed=None` usages which would make some test non-deterministic. Are these `None`s intentional or would we want to seed these too?

Reply:
FYI @galipremsagar